### PR TITLE
15 March homepage content programing update

### LINKED ISSUE #4515: 15 March homepage content programing update
For the Thursday, 15 March push, please update the AMO homepage programming in accord with what's [here](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#heading=h.2dl5oxy3ewcc).

@muffinresearch @diox @EnTeQuAk

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #4515 

I created the collections required for the home page on dev to make testing this possible. I guess we'll have to create them on stage as well once this gets pushed there. Is that something QA could do to support testing, or is it something we're required to do?

### CODE REVIEW

**Comment by bobsilverberg** [comment]
Oops, I need to update the test for the home page for this, which I forgot. Adding that now...

File: src/amo/components/Home/index.js [hunk 1]
```diff
@@ -252,8 +252,8 @@ export class HomeBase extends React.Component {
         <LandingAddonsCard
           addons={collections[2]}
           className="Home-FeaturedCollection"
-          header={i18n.gettext('Dynamic media downloaders')}
-          footerText={i18n.gettext('See more dynamic media downloaders')}
+          header={i18n.gettext('Trending extensions')}
+          footerText={i18n.gettext('See more trending extensions')}
           footerLink={
             `/collections/${COLLECTIONS_TO_FETCH[2].user}/${COLLECTIONS_TO_FETCH[2].slug}/`
```
Comment (tofumatt) [inline src/amo/components/Home/index.js]: Should trending extensions be a collection? I guess the programming update would say but I expected this to link to `/search/?sort=hotness&type=extension`

File: src/amo/components/HomeHeroBanner/index.js [hunk 2]
```diff
@@ -22,57 +22,58 @@ export class HomeHeroBannerBase extends React.Component<Props> {
...
-          linkTo="/addon/momentumdash/"
+          linkTo="/addon/autoformer/"
         >
-          <h3>{i18n.gettext('Momentum')}</h3>
+          <h3>{i18n.gettext('AutoFormer+')}</h3>
 
           <p>
-            {i18n.gettext(`Replace your new tab with a personal
-              dashboard—to-do lists, weather forecasts and more`)}
+            {i18n.gettext(`Fill out a lot of forms? This might help`)}
```
Comment (tofumatt) [inline src/amo/components/HomeHeroBanner/index.js]: This doesn't need to be a backtick-quoted string anymore, surprised eslint doesn't complain! I guess it's fine though.

File: src/amo/components/Home/index.js (hunk 1 above)
Comment (bobsilverberg) [reply src/amo/components/Home/index.js]: Good question. @devaneymoz, do you intend for `Trending extensions` to be populated via a collection, or can we just use the query as suggested above, an example of which would be https://addons-dev.allizom.org/en-US/firefox/search/?sort=hotness&type=extension ?

File: src/amo/components/HomeHeroBanner/index.js (hunk 2 above)
Comment (bobsilverberg) [reply src/amo/components/HomeHeroBanner/index.js]: I don't think eslint cares about this, but good point. It must have been a longer string before and I neglected to replace the backticks with single quotes. I'll change it.

**Comment by devaneymoz** [comment]
@bobsilverberg in this case the "Trending" collection is a hand-picked collection of emerging content.

**Comment by tofumatt** [comment]
That's a bit confusing as we also have a "trending" search filter. 😄

**Comment by bobsilverberg** [comment]
Thanks @devaneymoz. I agree with @tofumatt but whatever. You're the boss!

**Comment by tofumatt** [comment]
All good then!

**Comment by devaneymoz** [comment]
You guys make a good point about the 'trending' distinction between algorithm and curated! Let's please keep these changes for now but I'll work on finding a better editorial label to distinguish the two cases. 

Man I forgot what great hair Tony Danza has.

**Pull request merged**